python/rikai/torch/transforms/fasterrcnn_resnet50_fpn.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright 2021 Rikai Authors
```

Comment:
do we want this as part of the base library code that we'd want to keep compat for across torchvision versions etc?

Reply:
you meant maintaining something like `rikai-contrib` package?

Reply:
moved these files to `rikai.contrib` , we can later release this package separately.